4/typedefs.js

```diff
@@ -0,0 +1,214 @@
+const uuid = require('uuid/v1');
+
+const typeDefs = `
+
+  enum State {
+    DONE
+    UNDONE
+    IN_PROGRESS
+  }
+
+  enum Order {
+    DESC
+    ASC
+  }
+
+  type Todo {
+    id: ID!
+    message: String!
+    state: State
+    assignee: Assignee @relation(name: "ASSIGNED_TO", direction: "IN")
+  }
+
+  type Assignee {
+    id: ID!
+    name: String!
+  }
+
+  type Query {
+    todos: [Todo]
+    todo(id: ID!): Todo
+    todosOrderedByState(sortOrder: Order!): [Todo]
+    skipAndLimitTodos(skip: Int, limit: Int): [Todo]
+  }
+
+  type Mutation {
+    createAssignee(id: ID, name: String!): Assignee
+    createTodo(id: ID, message: String!, state: State): Todo
+    createRelationship(id: ID!, name: String!): Todo
+    deleteTodo(id: ID!): Todo
```

Comment:
You declare these mutations with a Todo return value, however you only return a boolean value